<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Result</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QLabel" name="lb_compute_time">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>120</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lb_inference_time">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Max FPS: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Compute Time: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lb_max_fps">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>200</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Inference Time: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="te_detect_res"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pb_start_detect">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_pause_detect">
           <property name="text">
            <string>pause</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_stop_detect">
           <property name="text">
            <string>stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Display</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_dispaly">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>72</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>no signal</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
   </widget>
   <widget class="QMenu" name="menunet">
    <property name="title">
     <string>Net</string>
    </property>
    <addaction name="action_custom_net"/>
    <addaction name="separator"/>
    <addaction name="action_yolov3_net"/>
    <addaction name="action_yolov4_net"/>
    <addaction name="actionyolov4fastest"/>
    <addaction name="action_yolobile_net"/>
    <addaction name="separator"/>
    <addaction name="actionuse_cuda"/>
   </widget>
   <widget class="QMenu" name="menudetect">
    <property name="title">
     <string>Detect</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_IMG_path"/>
    <addaction name="action_Video_path"/>
    <addaction name="action_ZED_CAM"/>
   </widget>
   <widget class="QMenu" name="menuTrack">
    <property name="title">
     <string>Track</string>
    </property>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menunet"/>
   <addaction name="menudetect"/>
   <addaction name="menuTrack"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_custom_net">
   <property name="text">
    <string>custom</string>
   </property>
  </action>
  <action name="action_yolov3_net">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov3</string>
   </property>
  </action>
  <action name="action_yolov4_net">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov4</string>
   </property>
  </action>
  <action name="actionyolov4fastest">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolov4_fastest_net</string>
   </property>
  </action>
  <action name="action_yolobile_net">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yolobile</string>
   </property>
  </action>
  <action name="actionuse_cuda">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>use cuda</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="action_IMG_path">
   <property name="text">
    <string>IMG</string>
   </property>
  </action>
  <action name="action_Video_path">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="action_ZED_CAM">
   <property name="text">
    <string>ZED CAM</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
